### PR TITLE
fix: Update frontend.mdx

### PR DESCRIPTION
**Pull request by tacurran**
@jfcurran  #291

### CODE REVIEW

**Comment by aeneasr** [comment]
@jfcurran please review this

File: src/pages/jobs/frontend.mdx [hunk 1]
```diff
@@ -24,41 +24,37 @@ Location: Munich, Germany
...
 - Experience in JavaScript, and one or more programming languages including but
-  not limited to Java, C/C++, Python or Go.
+  not limited to Java, C/C++, Rust, Python or Go.
 
 - Experience with front-end technologies and/or front-end frameworks.
 
 ### Preferred Requirements:
 
-- Experience developing user-facing software.
+- Experience developing graphical user face software
```
Comment (jfcurran) [inline src/pages/jobs/frontend.mdx:33]:
```suggestion
- Experience developing graphical user facing software
```

File: src/pages/jobs/frontend.mdx [hunk 2]
```diff
@@ -24,41 +24,37 @@ Location: Munich, Germany
...
 
-- Experience with the latest web standards.
+- Experience with the latest web standards
 
 - Experience with JavaScript and front-end development frameworks such as React,
   MaterialUI, etc.
 
-- Ability to project manage projects and prioritize tasks.
+- Ability to organise work, manage projects, and prioritize tasks, and
```
Comment (jfcurran) [inline src/pages/jobs/frontend.mdx:40]:
```suggestion
- Ability to organise work, manage projects, and prioritize tasks.
```

**Review by jfcurran — CHANGES_REQUESTED** [review]

**Comment by jfcurran** [comment]
I am collecting these changes and adding them to a different patch together with @tacurran - closing for now

**Pull request closed without merging**